Sources/OTel/OTelCore/Logging/Exporting/OTelConsoleLogRecordExporter.swift

```diff
@@ -34,6 +34,8 @@ struct OTelConsoleLogRecordExporter: OTelLogRecordExporter {
             ---
             """
         )
+        // No background work needed, but we'll keep the run method running until its cancelled.
+        await AsyncStream.makeStream(of: Void.self).stream.cancelOnGracefulShutdown().first { _ in true }
```

Comment:
Could we use ServiceLifecycle's top-level `gracefulShutdown()` method instead here?

https://github.com/swift-server/swift-service-lifecycle/blob/21cf72f3604f80aec767ceb367c9b91e2a95f504/Sources/ServiceLifecycle/GracefulShutdown.swift#L194

Something like `try? await gracefulShutdown()`.

Reply:
Oh, maybe. Let me see.